test/integration/tests/custom-context.test.ts

```diff
@@ -1,25 +1,50 @@
 import { PolarisServer } from '../../../src';
+import * as customContextFields from '../server/constants/custom-context-fields.json';
 import { initializeDatabase } from '../server/dal/data-initalizer';
 import { startTestServer, stopTestServer } from '../server/test-server';
 import { graphQLRequest } from '../server/utils/graphql-client';
-import * as request from './jsonRequestsAndHeaders/customHeaders.json';
+import * as customContextCustomFieldRequest from './jsonRequestsAndHeaders/customContextCustomField.json';
+import * as customContextInstanceMethodRequest from './jsonRequestsAndHeaders/customContextInstanceMethod.json';
+import * as customHeadersRequest from './jsonRequestsAndHeaders/customHeaders.json';
 
 let polarisServer: PolarisServer;
 
-beforeEach(async () => {
+beforeAll(async () => {
```

Comment:
why did you change it to beforeAll?
change it back to before each

Reply:
discussed on WhAtSaPp